### PR TITLE
Resomi fixes

### PR DESCRIPTION
**Pull request by Draxtheros**
The language was not included, nor were names set up generate properly based on the species language. Additionally, some clothes that were missing their in-world icon have been fixed.

### CODE REVIEW

File: code/modules/species/station/resomi.dm [hunk 1]
```diff
@@ -85,6 +85,17 @@
 		/datum/mob_descriptor/height = -2,
 		/datum/mob_descriptor/build = -1
 		)
+		
+	available_cultural_info = list(
```
Comment (PsyCommando) [inline code/modules/species/station/resomi.dm:89]: I'd add the other basic tags too. Just in case. Because culture tags are super finicky.

**Review by PsyCommando — CHANGES_REQUESTED** [review]
Looks good! Just add the other standard `available_cultural_info` for the other default tags. Because, the thing is very finicky, and it'll make it compatible with standard bay too.

File: code/modules/species/station/resomi.dm (hunk 1 above)
Comment (Draxtheros) [reply code/modules/species/station/resomi.dm:89]: It should be fine, the Human entry also only has the two, with the rest defaulting to whatever is in the base species.dm

Comment (PsyCommando) [reply code/modules/species/station/resomi.dm:89]: Alright!

**Review by PsyCommando — APPROVED** [review]

**Pull request merged**